File: java/Kafic/src/BezalkoholnoPice.java
public class BezalkoholnoPice extends Pice {
    protected boolean energetsko;

    public BezalkoholnoPice(String naziv, String brend, Ukus ukus) {
        super(naziv, brend, ukus);
    }

    public void opisiPice() {
        super.opisiPice();
        System.out.println("Energetysko: " + energetsko);
    }

    public  boolean isEnergetsko() {
        return energetsko;
    }

    public void setEnergetsko(boolean energetsko) {
        this.energetsko = energetsko;
    }
}

File: typescript/Kafic/BezalkoholnoPice.ts
import { Ukus } from './Ukus'
import { Pice } from './Pice'

export class BezalkoholnoPice extends Pice {
  protected energetsko: boolean;

  constructor(naziv: string, brend: string, ukus: Ukus) {
      super();
  }

  public opisiPice(): void {
      super.opisiPice();
      console.log("Energetysko: " + this.energetsko);
  }

  public isEnergetsko(): boolean {
      return this.energetsko;
  }

  public setEnergetsko(energetsko: boolean) : void {
      this.energetsko = energetsko;
  }
}
